cmd/root.go

```diff
@@ -167,7 +167,7 @@ func initConfig() {
 
 func checkUpdateAvailable(checkUpdateChan chan interface{}) {
 	defer close(checkUpdateChan)
-	updateAvailable, latestVersion, err := updates.UpdateAvailable("yext", "edward", common.EdwardVersion, filepath.Join(home.EdwardConfig.Dir, ".updatecache"), logger)
+	updateAvailable, latestVersion, err := updates.UpdateAvailable("yext", "edward", common.EdwardVersion, filepath.Join(home.EdwardConfig.Dir, ".cache"), logger)
```

Comment:
This seems a little generic, especially if we ever decide to add other caching.
Perhaps .cache/version would be a better option?

Reply:
`diskcache` expects directory (himself creates a file) aka `.cache/5872f43ir84yfgc3fg484f`

Reply:
Agree, fix.